ckan/requirements.txt

```diff
@@ -20,25 +20,26 @@ ckanext-googleanalyticsbasic==0.2.0
 -e git+https://github.com/ckan/ckanext-harvest.git@10e50a709b8ff472d7e4c7ffc880513f5cf1ef7f#egg=ckanext_harvest
 ckanext-metrics-dashboard==0.1.5
 -e git+https://github.com/ckan/ckanext-qa.git@1731b59d2bf82b06f7866c204b26eb7c6c9ea1f9#egg=ckanext_qa
--e git+https://github.com/ckan/ckanext-report.git@24ade58d126d9480286c7bd1a904f20186d83988#egg=ckanext_report
+-e git+https://github.com/ckan/ckanext-report.git@3588577f46d17e5f6ef163bb984d0e7016daef71#egg=ckanext_report
 ckanext-saml2auth @ git+https://github.com/keitaroinc/ckanext-saml2auth.git@9943fb22b61abdf5c9efdbb3f99dbc428ad53fbc
--e git+https://github.com/ckan/ckanext-spatial.git@17d5a341cf8f40b35b25df91a18ce72c31195ba3#egg=ckanext_spatial
+-e git+https://github.com/ckan/ckanext-spatial.git@9d9bc236952ab055f59d392304aad148fd945e75#egg=ckanext_spatial
```

Comment:
This introduced a new dependency, `geojson` (not sure if there are any major changes we should be aware about).